src/screens/tabs/Profile/screens/Settings/index.js

```diff
@@ -63,32 +65,35 @@ const Settings = () => {
     );
   };
 
-  const settingsItems = [
-    {
-      icon: '📓',
-      name: 'Contacts',
-      description: 'Add, edit, remove contacts.',
-      onPress: openContacts,
-    },
-    {
-      icon: '🗝',
-      name: 'Seed Phrase',
-      description: 'View your seed phrase & backup.',
-      onPress: openRevealSeedPhrase,
-    },
-    {
-      iconName: 'faceIdIcon',
-      name: 'Biometric Unlock',
-      description: 'Turn Biometrics on or off.',
-      onPress: openBiometricUnlock,
-    },
-    {
-      icon: '🔒',
-      name: 'Lock Account',
-      description: 'Lock your account and sign out.',
-      onPress: lockAccount,
-    },
-  ];
+  const settingsItems = useMemo(
+    () => [
+      {
+        icon: '📓',
+        name: t('settings.items.contacts.name'),
+        description: t('settings.items.contacts.desc'),
+        onPress: openContacts,
+      },
+      {
+        icon: '🗝',
+        name: t('settings.items.phrase.name'),
+        description: t('settings.items.phrase.desc'),
+        onPress: openRevealSeedPhrase,
+      },
+      {
+        iconName: 'faceIdIcon',
+        name: t('settings.items.biometric.name'),
+        description: t('settings.items.biometric.desc'),
+        onPress: openBiometricUnlock,
+      },
+      {
+        icon: '🔒',
+        name: t('settings.items.lock.name'),
+        description: t('settings.items.lock.desc'),
+        onPress: lockAccount,
+      },
+    ],
+    [],
+  );
```

Comment:
why useMemo here?

Reply:
because in every render of the function, a new object is created, and its not necessary

Reply:
Perfect bro!